test/replication/long_row_timeout.result

```diff
@@ -73,9 +75,9 @@ test_run:cmd('setopt delimiter ""');
 ---
 - true
 ...
-ok
+status
 ---
-- true
+- stopped
```

Comment:
You have status = 'stopped' in the result file. Doesn't look right.

Reply:
Right, corrected, thanks.